src/pages/Admin/Sidebar/SidebarOpener/SidebarOpener.tsx

```diff
@@ -11,7 +11,7 @@ export function SidebarOpener({ className = "", linkGroups }: Props) {
     <button
       type="button"
       onClick={open}
-      className={`flex items-center gap-2 py-5 px-6 dark:bg-blue-d6 border-b border-prim font-bold text-sm text-orange ${className}`}
+      className={`flex items-center gap-2 py-5 px-6 dark:bg-blue-d6 border-b border-prim font-bold text-sm text-orange h-2 ${className}`}
```

Comment:
@SovereignAndrey, maybe decreasing   `py-5` can accomplish the same - quite unusual to restrict height while padding is large (overflows height)

Reply:
Will revert this change, as I can't seem to reproduce it now. If we see it again we can fix it then. :+1: